vllm/model_executor/models/deepseek_v2.py

```diff
@@ -290,7 +310,7 @@ def forward(
         kv_a, _ = latent_cache.split(
             [self.kv_lora_rank, self.qk_rope_head_dim], dim=-1)
         latent_cache = latent_cache.unsqueeze(1)
-        kv_a = self.kv_a_layernorm(kv_a.contiguous())
+        kv_a = self.kv_a_layernorm(kv_a.contiguous().unsqueeze(0)).squeeze(0) # w/a of SW-208144
```

Comment:
This should be under if_hpu

Reply:
will open a new PR

Reply:
https://github.com/HabanaAI/vllm-fork/pull/516